lib/referee-sinon.test.js

```diff
@@ -68,28 +68,30 @@ describe("referee-sinon", function () {
       "notCalled",
       "pass",
     ];
+    // eslint-disable-next-line mocha/no-setup-in-describe
```

Comment:
Maybe we should consider removing this rule in a future version of our linting rules.

Reply:
Either that, or write our tests in a different way